live-build/misc/ansible-roles/appliance-build.install-delphix-s3-packages/tasks/main.yml

```diff
@@ -56,10 +57,15 @@
   args:
     chdir: "{{ tempdir.path }}"
 
-- apt:
-    deb: "{{ item }}"
-  with_fileglob:
-    - "{{ tempdir.path }}/*.deb"
+#
+# We use apt-get here instead of ansible's apt module to install all the debs
+# at the same time, so that any dependencies between provided packages are
```

Comment:
Can you try to expand on this comment a bit, and more clearly define what "dependencies between provided packages" means? perhaps with an example.. I think I understand, but having an example would probably make things more clear, especially months later when I forget how this works..